src/gcp/cloudbuild.ts

```diff
@@ -158,10 +158,18 @@ export async function fetchLinkableRepositories(
   projectId: string,
   location: string,
   connectionId: string,
-): Promise<LinkableRepositories> {
+): Promise<Repository[]> {
   const name = `projects/${projectId}/locations/${location}/connections/${connectionId}:fetchLinkableRepositories`;
-  const res = await client.get<LinkableRepositories>(name);
-  return res.body;
+  let pageToken = "";
```

Comment:
do you mind reverting this file? this will be addressed very shortly in https://github.com/firebase/firebase-tools/pull/6764/files#diff-2d40d8ede700bdabb5e1e6f7141916628fd1dfc44a55e68582be14ad1cc44398

we might as well avoid the merge conflict now

Reply:
Yes apologies if it was unclear that I was including this in my PR